Chapter-4-applying-tactical-domain-driven-design/README.adoc

```diff
@@ -40,29 +77,238 @@ IMPORTANT: It makes no sense to consider the Domain Model in typical CRUD module
 Business requirements changed a lot in comparison to Chapter 3:
 
 1. _Contract_ can still be prepared but does not have the force of law - we treat it as a draft.
-2. After the _Contract_ is signed, _Binding Contract_ is created. It has the force of law and binds the customer with us.
-3. After three months, at any time, the customer can terminate _Binding Contract_ without any penalty.
-4. It is possible to attach an _Annex_ to the existing _Binding Contract_. This way, the customer can extend the contract for another year without preparing a new _Contract_ for him.
-5. Annex can only be attached if _Binding Contract_ is active - has not yet expired or was not terminated.
+2. After the _Contract_ is signed, a __Binding Contract__ is created. It has the force of law and binds the customer with us.
+3. After three months, at any time, the customer can terminate __Binding Contract__ without any penalty.
+4. It is possible to attach an _Annex_ to the existing __Binding Contract__. This way, the customer can extend the contract for another year without preparing a new _Contract_ for him.
+5. Annex can only be attached if a __Binding Contract__ is active - has not yet expired or was not terminated.
 
-image::Assets/flow.jpg[]
+image::Assets/flow.png[]
 
 === Main assumptions
 
 The assumptions remain unchanged to keep the environment comparable to the previous step.
 
 === Solution
 
-==== Overview
+== Overview
 
-In this step we do not change the project structure of the application. We focus only on refactoring the code of _Contracts_ microservice.
+In this step, we do not change the project structure of the application. We focus only on implementing new features and refactoring the code of the _Contracts_ microservice.
 
 We introduce elements like:
 
-- Aggregates
-- Entities
-- Value Objects
-- Domain Events
+- <<aggregate-root,Aggregates>>
+- <<entity,Entities>>
+- <<value-object,Value Objects>>
+- <<domain-events,Domain Events>>
+
+=== Contracts Bounded Context Tactical Design Analysis
+
+image::Assets/design-level-event-storming.png[design level event storming]
```

Comment:
I will fix this image in the next branch after we are ready with this one. From the one that looks similar to the process level, I would switch it to a horizontal one (better fitting from the perspective of ES design level)

Reply:
I'm happy to see how it will looks like